less/components/menu.less

```diff
@@ -60,6 +60,55 @@
         padding-bottom:0;
     }
 }
+
+@media (max-width: @screen-xs-max) {
+    #main-menu {
+        .dropdown-menu {
```

Comment:
We can probably combine this line with the previous as there is no classes that target the main-menu other than the ones under dropdown-menu.